README.md

```diff
@@ -3,3 +3,13 @@
 [![Build Status](https://github.com/rust-osdev/x86_64/workflows/Build/badge.svg)](https://github.com/rust-osdev/x86_64/actions?query=workflow%3ABuild) [![docs.rs](https://img.shields.io/badge/docs.rs-documentation-green.svg)](https://docs.rs/x86_64)
 
 Support for x86_64 specific instructions (e.g. TLB flush), registers (e.g. control registers), and structures (e.g. page tables).
+
+## Feature Flags
```

Comment:
I think "Cargo Features" or "Crate Features" would make it more clear.

Reply:
I went with `Crate Feature Flags` as you can set these manually without cargo also.